atlas-engine/ci_tools
# :sparkles: Add new command to replace dist-tags with real versions

## Changes

1. Add a new command "replace-dist-tags-with-real-versions"
2. Add npm-dependency-related functions and tests
3. Add an optional "pathspec" parameter to "isDirty" (git-related function)

## Issues

## How to test the changes

1. Install or link the CI tools globally
2. Navigate to an AtlasEngine repository
3. Use dist tags (e.g. "alpha", "beta" or "feature~xyz") for atlas-engine-related packages
4. Run `ci_tools replace-dist-tags-with-real-versions`
5. Check if dist tags were replaced with real versions
